src/pages/Home/index.tsx

```diff
@@ -1,15 +1,20 @@
 import { Container } from 'react-bootstrap';
 import { useEffect, useState } from 'react';
 import { OrderProps, BobaProps } from 'types/common/main';
-import { bobaList, toppingsList, milkList } from 'assets/sampleBobaAPI';
+import { toppingsList, milkList } from 'assets/sampleBobaAPI';
 import BobaContainer from 'components/BobaContainer';
 import SearchBar from 'components/SearchBar';
 import CustomizeModal from 'components/Modal';
+import Loading from 'components/Loading';
 import BobaModalForm from './CustomizeBobaModalBody';
 import Button from 'react-bootstrap/Button';
+import useApi from 'hooks/API';
 
 const Homepage = ({ order, setOrder }: OrderProps): JSX.Element => {
-  const [filteredBobaList, setFilteredBobaList] = useState<BobaProps[]>(bobaList);
+  const { data, error, loading } = useApi(
+    `${process.env.REACT_APP_BOBA_FETCH}/boba_list` as string,
```

Comment:
split params into url and end point as discussed
```useApi(`${process.env.REACT_APP_BOBA_FETCH}/boba_list` as string);
to
useApi(url=process.env.REACT_APP_BOBA_FETCH, endpoint="/boba_list");
[3:55](https://burmesepython.slack.com/archives/D010ZD6LZ7A/p1681599315523929)
you append those two back inside the useApi
[3:55](https://burmesepython.slack.com/archives/D010ZD6LZ7A/p1681599346073889)
then you setData with
if(endpoint="/bobalist"){
   setData(bobalist)
}elseif(endpoint="/toppings"){
   setData(toppingList)
```

Reply:
got it , will be updated